core/cas-server-core-util-api/src/test/java/org/apereo/cas/util/logging/DefaultLogMessageSummarizerTests.java

```diff
@@ -0,0 +1,25 @@
+package org.apereo.cas.util.logging;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+/**
+ * This is {@link DefaultLogMessageSummarizerTests}.
+ *
+ * @author Chaim
+ * @since 7.0.0
+ */
+@Tag("Utility")
+@Slf4j
+class DefaultLogMessageSummarizerTests {
```

Comment:
Please make sure your test classes follow the same convention as every other class in the codebase and also run a style check on your changes.